tests/meta/transfer/test_external_data.py

```diff
@@ -174,6 +175,30 @@ def test_add_transfer_to_housekeeper(
     )
 
 
+def test_get_available_samples(
+    analysis_store_trio,
+    customer_id: str,
+    external_data_api: ExternalDataAPI,
+    sample_obj: models.Sample,
+    ticket_nr: int,
+    tmpdir_factory,
+):
+    # GIVEN that the root directory does not contain any correct folders
+    # THEN the function should return an empty list
+    transferred_samples = external_data_api.get_available_samples(
+        folder=Path("."), ticket_id=ticket_nr
+    )
+    assert transferred_samples == []
```

Comment:
```suggestion
    # GIVEN that the root directory does not contain any correct folders

    available_samples = external_data_api.get_available_samples(
        folder=Path("."), ticket_id=ticket_nr
    )
    
     # THEN the function should return an empty list
    assert transferred_samples == []
```
Maybe we should point the Path(".") to a place within tests that we know never will be correct

Reply:
Sure! What about an empty temporary directory?

Reply:
Sounds good